.ruby-version

```diff
@@ -1 +1 @@
-2.2.3@mentorhub
+2.3.0@mentorhub
```

Comment:
No, when we initially had a discussion we figured out that the upgrade is not causing any problems to our current code base. Hence we decided to use latest.